app/src/main/java/com/example/inharmony/LoginActivity.java

```diff
@@ -128,6 +133,14 @@ public void run() {
                             String email = service.getMe().email;
                             String id = service.getMe().id;
                             List<String> genreList = service.getSeedsGenres().genres;
+                            Map<String, Object> map = new HashMap<>();
```

Comment:
a more descriptive name would make this more readable - also where is `map` used?